### DISCUSSION TITLE
24w20a piston sounds

### DISCUSSION THREAD

**Discussion by BrianGLHF** [BedrockTweaks/Files#85] [Ideas]
Would like to be able to have a pack to use the new piston sounds from 24w20a snapshot that were reverted in the following snapshot.
Thank you!

**Comment by SquatchHunter** [comment]
_I thought I replied to this suggestion aswell_
Community Suggestions are currently slated for post addons release

in saying that I already have this pack made and sitting on my local system pending an approving nod from DrAv

**Reply by BrianGLHF** [reply]
did this ever make it on the website? I tried to find it today but I might have missed it.
thank you :)

**Reply by SquatchHunter** [reply]
its not on the site, we still havent released addons
in saying that I do have the pack made locally and ready to go if DrAv will accept it :)
